src/kasn1/pkcs.rs

```diff
@@ -90,3 +90,37 @@ pub const HMAC_SHA_256_ALG: AlgorithmIdentifier<'_> = AlgorithmIdentifier {
     oid: asn1::DefinedByMarker::marker(),
     params: AlgorithmParameters::HmacWithSha256(Some(())),
 };
+
+/// RSA Algorithm Identifier, typically used for Identifying a private or
+/// public key (for example in PrivateKeyInfo
```

Comment:
```suggestion
/// public key (for example in PrivateKeyInfo)
```
same on the following cases

Reply:
good catch